README.md

```diff
@@ -21,6 +21,30 @@ For everything else, the targets can be created manually after the dependency ha
 - [CPM: An Awesome Dependency Manager for C++ with CMake](https://medium.com/swlh/cpm-an-awesome-dependency-manager-for-c-with-cmake-3c53f4376766)
 - [CMake and the Future of C++ Package Management](https://ibob.github.io/blog/2020/01/13/cmake-package-management/)
 
+## Full CMakeLists Example
+
+```cmake
+cmake_minimum_required(VERSION 3.14 FATAL_ERROR)
+
+# create project
+project(MyProject)
+
+# add executable
+add_executable(main main.cpp)
+
+# add dependencies
+include(cmake/CPM.cmake)
+
+CPMAddPackage("gh:fmtlib/fmt#7.1.3")
+CPMAddPackage("gh:nlohmann/json@3.10.5")
+CPMAddPackage("gh:catchorg/Catch2@2.5.0")
+
+# link dependencies
+target_link_libraries(main fmt nlohmann_json::nlohmann_json Catch2)
```

Comment:
why is `nlohmann_json` namespaced and the others are not?

Reply:
Good point, I wasn't aware that the other libraries also create namespaced targets.

```suggestion
target_link_libraries(main fmt::fmt nlohmann_json::nlohmann_json Catch2::Catch2)
```